data.sql

```diff
@@ -12,3 +12,42 @@ VALUES ('Agumon', date '2020-02-03', 10.23, true, 0),
 ('Boarmon', '2005-06-07', -20.4, true, 7),
 ('Blossom', '1998-10-13', 17, true, 3),
 ('Ditto', '2022-05-14', 22, true, 4);
+
+-- Day 3
+INSERT INTO owners (full_name, age)
+VALUES
+('Sam Smith', 34),
+('Jennifer Orwell', 19),
+('Bob', 45),
+('Melody Pond', 77),
+('Dean Winchester', 14),
+('Jodie Whittaker', 38);
+
+INSERT INTO species(name)
+VALUES
+('Pokemon'),
+('Digimon');
+
+UPDATE animals
+SET species_id = '2'
+WHERE name LIKE '%mon';
+
```

Comment:
- [ ] You have done the first UPDATE and u used the hardcoded value for species_id which is ok, but I cannot find the UPDATE for all other animal species.  Check requirement to understand better:
`if an animal's name ends with mon it should have Digimon as a species_id, all other animals should have Pokemon instead`
I think something like this can work
`UPDATE animals SET species_id = (SELECT id from species WHERE name = 'Pokemon') WHERE species_id IS NULL; `